piptools/_compat/pip_compat.py

```diff
@@ -36,6 +38,26 @@ def parse_requirements(
         yield install_req_from_parsed_requirement(parsed_req, isolated=isolated)
 
 
+def install_req_from_line(
+    name: str,
+    comes_from: str | InstallRequirement | None = None,
+    base_package: str | None = None,
+    base_dir: str | None = None,
+) -> InstallRequirement:
```

Comment:
I'm wondering if that makes sense to move this function to `utils` module and perhaps give it a different name to not confuse it with the original `install_req_from_line` which is used in other places?

Reply:
What name could you suggest? install_req_from_base_dir?

Reply:
I came up with https://github.com/q0w/pip-tools/pull/1

Reply:
Nice, merged